buildstockbatch/aws/awsbase.py

```diff
@@ -1,8 +1,11 @@
 import logging
+from botocore.config import Config
 
 
 logger = logging.getLogger(__name__)
 
+boto_client_config = Config(retries={"max_attempts": 5, "mode": "standard"})
```

Comment:
I wonder what prompted this. Did things started requiring retries?

Reply:
This retries on throttling errors automatically by switching from legacy to standard mode. See [docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html#standard-retry-mode). I did it when removing the AwsRetry dependency because it wasn't being updated and didn't work right on Windows.